README.rst

```diff
@@ -124,6 +124,9 @@ a test name, and insert into it some code. The tests will run pylint
 against these modules. If the idea is that no messages now occur, then
 that is fine, just check to see if it works by running ``scripts/test.sh``.
 
+You can pass any pytest commaned line argument to ``scripts/test.sh``.
```

Comment:
I don't see pytest used here so maybe using the wrong wording (also in commit log) ? 

`commaned` -> `command`

Reply:
`test.sh` calls [test_func.py](https://github.com/PyCQA/pylint-django/blob/master/pylint_django/tests/test_func.py) which than calls pytest and passes its arguments to pytests invocation.

I have fixed the typo.